library/build.gradle

```diff
@@ -127,7 +127,8 @@ dependencies {
   api deps.rxjava2
   api deps.rxandroid2
   api deps.rxkotlin2
-  implementation deps.supportannotations
+  implementation 'androidx.annotation:annotation:1.0.1'
```

Comment:
I this project, I define all dependencies in the main top-level `build.gradle` file and it should be done this way here too.